front/src/modules/timesStops/styles/_timesStopsDatasheet.scss

```diff
@@ -1,31 +1,61 @@
 .time-stops-datasheet {
+  padding-bottom: 16px;
+  border-radius: 10px;
+  box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1);
```

Comment:
The shadow box on the mockup is a little different: 
box-shadow: 0px 1px 0px rgba(255, 255, 255, 1) inset, 0px 4px 7px -3px rgba(255, 171, 88, 0.17), 0px 2px 4px rgba(0, 0, 0, 0.22);

Reply:
Good catch thanks! How did you get the exact box-shadow css? I couldn't find it on the mockup.

Reply:
right click and 'elevation' :) 
<img width="671" alt="image" src="https://github.com/user-attachments/assets/9bbb78b1-0fae-4fe5-9763-766ced0c266b">